src/program/command_mode.rs

```diff
@@ -310,6 +335,9 @@ impl<'a> Program<'a> {
                     return Ok(());
                 }
                 self.newglob(&arguments);
+                if self.paths.dest_folder.ends_with("keep") {
+                    self.paths.dest_folder = self.paths.current_dir.join("keep");
```

Comment:
In your above example that's how it should function currently. 

The only instance it would change in newglob is if the user hasn't modified the default destination folder. If that's the case we should maintain the invariant, stated in help/README, that the default folder should be `./keep`

I think that would make sense, for instance running riv in `/riv/src/program` initially and going to `~/Pictures` and hitting `move` it'd be really weird if we moved the image to `/riv/src/program/keep` rather than `~/Pictures/keep` I personally would think that's unexpected.